bin/improve_assembly

```diff
@@ -82,12 +95,31 @@ USAGE
 
 $debug           ||= 0;
 $insert_size     ||= 250;
+
 $scaffolder_exec ||= '/software/pathogen/external/apps/usr/local/SSPACE-BASIC-2.0_linux-x86_64/SSPACE_Basic_v2.0.pl';
 $gapfiller_exec  ||= '/software/pathogen/external/apps/usr/local/GapFiller_v1-10_linux-x86_64/GapFiller.pl';
-$scaffolder_exec = 'SSPACE_Basic_v2.0.pl' unless(Bio::AssemblyImprovement::Validate::Executable->new()->does_executable_exist($scaffolder_exec));
-$gapfiller_exec = 'GapFiller.pl' unless(Bio::AssemblyImprovement::Validate::Executable->new()->does_executable_exist($gapfiller_exec));
+$abacas_exec     ||= '/software/pathogen/internal/prod/bin/abacas.pl';
+
+# Validate the executables
+if(Bio::AssemblyImprovement::Validate::Executable->new()->does_executable_exist($scaffolder_exec)){
+	$scaffolder_exec = 'SSPACE_Basic_v2.0.pl';
+} else {
+	die "$scaffolder_exec does not exist.";
+}
+
+if(Bio::AssemblyImprovement::Validate::Executable->new()->does_executable_exist($gapfiller_exec)){
```

Comment:
Does this do what you expect?  If -g isn't given, line 100 makes `$gapfiller_exec  ||= '/software/pathogen/external/apps/usr/local/GapFiller_v1-10_linux-x86_64/GapFiller.pl';`.  In https://github.com/sanger-pathogens/assembly_improvement/blob/master/lib/Bio/AssemblyImprovement/Validate/Executable.pm it looks for a '/' (which this has) and returns 1.  You therefore set `$gapfiller_exec = 'GapFiller.pl';` so the default above can never be applied (I think).